rules/power-automate-flows-convert-timezone/rule.md

```diff
@@ -0,0 +1,40 @@
+---
+type: rule
+title: Do you know how to use correct timezone in your Power Automate flows?
+uri: power-automate-flows-convert-timezone
+authors:
+  - title: Aman Kumar
+    url: https://ssw.com.au/people/aman-kumar
+related: []
+created: 2023-10-12T12:33:21.000Z
+guid: f20c1d67-fbd5-4fa6-99ba-58b4c8f53fd7
+---
+
+When working with Power Automate flows, there are instances where you need to use or compare the time in your flow, by default, it is set to UTC.
+
+However, there are situations where using your local timezone might be more appropriate, especially for tasks or events that are specific to a particular region. It's important to choose the appropriate timezone based on the context and requirements of your task or system.
+
+<!--endintro-->
+
+Here is a bad example of comparing Datetime in default timezone in PA Flows:
+
+::: bad
+![Figure: Bad example - You are comparing the UTC with UTC](comparing-timedate-without-converting.png)
+:::
+
+- ❌ **Complex Time Zone Calculations**: Handling time zone conversions manually can become cumbersome, especially when dealing with events and meetings across multiple time zones.
```

Comment:
Hey @amankumarrr 

Can you follow https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists in these list items?

Reply:
Done - removed